docs/cmd/kn_channel.md

```diff
@@ -31,4 +31,5 @@ kn channel COMMAND
 * [kn channel delete](kn_channel_delete.md)	 - Delete a channel
 * [kn channel describe](kn_channel_describe.md)	 - Show details of a channel
 * [kn channel list](kn_channel_list.md)	 - List channels
+* [kn channel list-types](kn_channel_list-types.md)	 - List channel types
```

Comment:
While just thinking about this, why not just `kn channel types` ? I know this violates the <noun> <verb> schema, but it looks imo nice (same for `kn source list-types`). At the end "list-types" is *also* not a verb but an artificial word construct.
wdyt @navidshaikh @maximilien ?

Reply:
I liked that. I'd keep it `list-types` for this PR and change for source and channel together in a subsequent PR (keeping list-types as an alias for source). cc @Kaustubh-pande